src/pages/public_registration/PublishingLogPreview.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { Ticket } from '../../types/publication_types/ticket.types';
+import { Registration } from '../../types/registration.types';
+import { toDateString } from '../../utils/date-helpers';
+import { generateSimplePublishingLog } from '../../utils/log/logFactory';
+import { StyledStatusMessageBox } from '../messages/components/PublishingRequestMessagesColumn';
+
+interface PublishingLogPreviewProps {
+  registration: Registration;
+  tickets: Ticket[];
+}
+
+export const PublishingLogPreview = ({ registration, tickets }: PublishingLogPreviewProps) => {
+  const { t } = useTranslation();
+  const logEntries = generateSimplePublishingLog(registration, tickets, t);
+
+  if (logEntries.length === 0) {
+    return null;
+  }
+
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.25rem', mb: '0.5rem' }}>
+      {logEntries.map((entry, index) => (
+        <StyledStatusMessageBox key={index} sx={{ bgcolor: 'publishingRequest.main' }}>
```

Comment:
VIl ikke `key={index}` gi en warning i konsollen? Har ikke testet, men har fått klager på det før. Vet vi bruker det andre plasser i appen og da

Reply:
Greit å bruke index her ja. Men om elementene kan bytte plass vil det gi bug da, siden det som var key=2 kan bli key=3 som er i konflikt med den forrige som var key=3 osv. Men så lenge de ikke bytter plass er det greit.
Så det beste er alltid å bruke en id på entries, men i dette tilfellet var dato det beste som var der, og date for publisere metadata og publisering av første fil kan jo fort være samme, og dermed duplisert, dermed tenkte jeg det var best å bare bruke en enkel indeks i stedet